src/com/mambu/apisdk/services/LoansService.java

```diff
@@ -534,6 +534,24 @@ public boolean patchLoanAccount(LoanAccount loan) throws MambuApiException {
 
 		return serviceExecutor.execute(getAccountTransactions, accountId, paramsMap);
 	}
+	
+	/**
+	 * Requests a list of loan transactions for a custom view, limited by offset/limit

```

Comment:
Since this is a convenience method, please document it as such and delegate the call to the existing getLoanTransactionsByCustomView() method with the offset and limit parameter values being null. Thanks!